_schemaV2.graphql

```diff
@@ -701,7 +701,6 @@ type Artist implements Node & Searchable & EntityWithFilterArtworksConnectionInt
   isFollowed: Boolean
   isPublic: Boolean
   isShareable: Boolean
-  isTargetSupply: Boolean
```

Comment:
@damassi I don't know why this change is appearing in the v2 schema. I saw this after I copy-pasted the new Gravity schema into `gravity.graphql` and ran `yarn dump:local`. Please let me know if removing this is a problem or expected.

P.S. I do have `ENABLE_CONSIGNMENTS_STITCHING=true` in my .env file.

Reply:
For Artsy employees, this was discussed in [a Slack thread](https://artsy.slack.com/archives/CS3H1K4BD/p1586446723395200.). The TL;DR is that this field used to exist as a top-level field in the "artist" type, as well as nested under a "target supply" type, and was removed as a top-level field. We missed a step somewhere and it wasn't removed from the V2 schema until this PR.